src/components/Article.jsx

```diff
@@ -1 +1,38 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({
+  title = '',
+  date,
+  text = '',
+}) {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title is-2">React Article</h1>
+
+        <div className="box">
+          <h2 className="title" data-cy="title">
+            {title}
+          </h2>
+
+          <p className="subtitle">
+            <time dateTime={dateISO} data-cy="date">
+              {dateLocaleString}
+            </time>
+          </p>
```

Comment:
You can use your consts right in <time> tag. Like this 
```suggestion
  
  return (
    <div className="section">
      <div className="container">
        <h1 className="title is-2">React Article</h1>

        <div className="box">
          <h2 className="title" data-cy="title">
            {title}
          </h2>

          <p className="subtitle">
            <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
              {date.toLocaleDateString('en-US', {
    						year: 'numeric',
    						month: 'long',
   						  day: 'numeric',
 							})}
            </time>
          </p>
```
and delete your consts. But I think your solution is good)

Reply:
Better to keep this solution. 
Such calculation looks better in the separate variables, easier in managing later and better readability